pyclient/confluo/rpc/rpc_service-remote

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env python
```

Comment:
This file should not be checked in.

Reply:
running genall.sh resulted in this; were there supposed to be no changes?

Reply:
Yes I know it generates this file, but we don't want to check it in. Ideally the genall.sh file should delete it explicitly -- I can add that change, but can you add a commit to remove the file?

Reply:
sounds good; are there any other files that shouldn't be checked in?

Reply:
No, that's the only one.